config.sh

```diff
@@ -27,6 +27,35 @@ BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.15
 BROTLI_VERSION=1.0.9
 
+function fuse_macos_intel_arm64 {
+    local wheelhouse=$(abspath ${WHEEL_SDIR:-wheelhouse})
+    local py_osx_ver=$(echo ${MB_PYTHON_OSX_VER} | sed "s/\./_/g")
+    mkdir -p tmp_fused_wheelhouse
+    for whl in $wheelhouse/*.whl; do
+       if [[ "$whl" == *macosx_${py_osx_ver}_x86_64.whl ]]; then
+           whl_base=$(sed "s/macosx_${py_osx_ver}_x86_64.whl//" <<< $whl)
+           if [[ -f "${whl_base}macosx_11_0_arm64.whl" ]]; then
+               delocate-fuse $whl "${whl_base}macosx_11_0_arm64.whl" -w tmp_fused_wheelhouse
+               mv tmp_fused_wheelhouse/$(basename $whl) $wheelhouse/$(basename ${whl_base})macosx_${py_osx_ver}_universal2.whl
+               rm $whl
+           fi
+       fi
+    done
+}
+
+function wrap_wheel_builder {
+    if [[ "${PLAT:-}" == "universal2" ]]; then
+        (macos_arm64_cross_build_setup && $@)
+        rm -rf *-stamp
+        (macos_intel_native_build_setup && $@)
+        fuse_macos_intel_arm64
+    elif [[ "${PLAT:-}" == "arm64" ]]; then
+        (macos_arm64_cross_build_setup && $@)
+    elif [[ "${PLAT:-}" == "x86_64" ]]; then
+        (macos_intel_native_build_setup && $@)
+    fi
+}
```

Comment:
There is debate about this in https://github.com/multi-build/multibuild/pull/495

Reply:
I've pushed a commit as an alternative to this, making use of https://github.com/multi-build/multibuild/pull/502